index.bs

```diff
@@ -582,37 +582,34 @@ interface ML {
 };
 </script>
 
+### Permissions Policy Integration ### {#permissions-policy-integration}
+
+This specification defines a <a>policy-controlled feature</a> identified by the
+string "<code><dfn data-lt="webnn-feature">webnn</dfn></code>".
+Its <a>default allowlist</a> is <code>'self'</code>.
+
+### The {{ML/createContext()}} method ### {#api-ml-createcontext}
 The {{ML/createContext()}} method steps are:
-1. If [=this=]'s [=relevant global object=]'s [=associated Document=] is not [=allowed to use=] the [=webnn-feature|webnn=] feature, then throw a "{{SecurityError!!exception}}" {{DOMException}} and abort these steps.
-1. Let |promise| be [=a new promise=].
+1. If [=this=]'s [=relevant global object=]'s [=associated Document=] is not [=allowed to use=] the [=webnn-feature|webnn=] feature, return [=a new promise=] [=rejected=] with a "{{SecurityError}}" and abort these steps.
+1. Return [=a new promise=] |promise| and run the following steps [=in parallel=].
 1. Let |context| be a new {{MLContext}} object.
-1. Switch on the method's first argument:
+1. Let |options| be the first argument.
+1. Switch on |option|:
     <dl class=switch>
-    <dt>{{MLContextOptions}}
+    <dt>{{MLContextOptions}} or {{undefined}}
     <dd>Set |context|.{{[[contextType]]}} to [=default-context|default=].
-    <dd>Set |context|.{{[[deviceType]]}} to the value of {{MLContextOptions}}'s {{deviceType}}.
-    <dd>Set |context|.{{[[powerPreference]]}} to the value of {{MLContextOptions}}'s {{powerPreference}}.
+    <dd>Set |context|.{{[[deviceType]]}} to the value of |option|'s {{deviceType}} or "[=device-type-cpu|cpu=]".
```

Comment:
s/option/options

Reply:
Thanks, fixed (will be committed later).